bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -41,28 +57,57 @@
  * @author Markus Rathgeb - Initial contribution and API
  * @author Mark Herwege - Added methods to find broadcast address(es)
  * @author Stefan Triller - Converted to OSGi service with primary ipv4 conf
+ * @author Gary Tse - Network address change listener
  */
 @Component(configurationPid = "org.eclipse.smarthome.network", property = { "service.pid=org.eclipse.smarthome.network",
         "service.config.description.uri=system:network", "service.config.label=Network Settings",
-        "service.config.category=system" })
+        "service.config.category=system", "service.config.network.interface_poll.cron_expression=43 0/2 * * * ?" })
 @NonNullByDefault
 public class NetUtil implements NetworkAddressService {
 
     private static final String PRIMARY_ADDRESS = "primaryAddress";
     private static final String BROADCAST_ADDRESS = "broadcastAddress";
+    private static final String INTERFACE_POLL_CRON_KEY = "service.config.network.interface_poll.cron_expression";
+    private static final String THREAD_POOL_NAME = NetUtil.class.getSimpleName();
```

Comment:
Please use "common" as a name, see my comments below. I have discussed it with @kaikreuzer and @htreu. At some point we should factor out these pool names into global constants though...